org.hdfgroup.HDFCompass.yaml

```diff
@@ -0,0 +1,104 @@
+id: org.hdfgroup.HDFCompass
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: HDFCompass
+finish-args:
+  - --socket=x11
+  - --device=all
+  - --share=ipc
+  - --filesystem=home
+
+modules:
+  - shared-modules/python2.7/python-2.7.json
```

Comment:
it really depends on python2?

Reply:
And this was a pain to package due to it.